proposals/2025-06-23-tsc-voting-procedures.md

```diff
@@ -0,0 +1,177 @@
+- Allow for discussion and clarification
+
+### Step 2: Call for Vote
+- TSC Chair or designee calls for vote
+- Confirm [quorum](https://github.com/hiero-ledger/hiero/blob/main/technical-charter.md?plain=1#L206) is present
+
+### Step 3: Voting
+- Members vote verbally or via meeting platform
+- Record each member's vote
+
+### Step 4: Resolution
+- Tally votes and announce result
+- Document in meeting minutes with vote counts
+
+## Voting Process for Asynchronous Votes
+
+### Step 1: Discussion (Optional)
+
+- Create a GitHub Discussion for complex proposals requiring discussion
+- Label with "tsc-discussion"
+- No formal timeline - continue until consensus on proposal language
+
+### Step 2: Proposal Creation
+
+1. Create a new file in `proposals/` directory following the template
+2. File naming convention: `YYYY-MM-DD-brief-description.md`
+3. Open a Pull Request with the finalized proposal
+4. **Proposal is immutable once PR is opened** - no changes allowed
+5. Link to discussion issue if one exists
+
+### Step 3: Voting Period
+
+- **Duration**: 14 calendar days maximum
+- **Purpose**: Voting only (discussion happens in issue, not PR)
+- **Voting Method**:
+  - Approve PR = "Yes" vote
+  - Request Changes = "No" vote
+  - No response = Abstention
+- **Comments**: Brief rationale only - substantive discussion in linked issue
+
+### Step 4: Resolution
+
+- **Passing Threshold**:
+  - Standard decisions: >50% of eligible voting members
+  - Charter-specified decisions: As defined in charter (e.g., 2/3 for license exceptions)
+- **Early Resolution**: Vote concludes immediately when:
+  - More than 50% of eligible members vote "Yes" (proposal passes)
+  - More than 50% of eligible members vote "No" (proposal fails)
+  - Mathematically impossible to change outcome
+- **Quorum**: Minimum participation of 50% of eligible voting members
+- **Recording**: 
+  - If passed: Merge PR, then create follow-up PR to add voting record
+  - If failed: Close PR with comment summarizing results
+
+## Voting Eligibility
+
+### Prerequisites
+
+- All TSC members must have a GitHub account to participate in voting
+
+### Active Voting Members
+
+- All TSC members listed in README.md
+- Have not missed the required number of consecutive TSC meetings defined in the [charter](https://github.com/hiero-ledger/hiero/blob/main/technical-charter.md?plain=1#L171)
+- Have GitHub account with appropriate repository access
+
+### Eligibility Tracking
+
+- Maintain eligibility status in `tsc-voting-eligibility.md`
+- Update after each meeting
+- Include:
+  - Member name
+  - GitHub username
+  - Voting status (active/suspended)
+  - Last meeting attended
+
+## Special Voting Scenarios
+
+### Tied Votes
+
+- **Resolution**:
+  - If tied: Proposal fails
+
+## Templates
+
+### Proposal Template
+
+See [proposal-template.md](./proposal-template.md) for the standard proposal format. This voting procedures proposal itself follows this template.
```

Comment:
We should document whether every section in the template is required or if all/some of the sections are optional

Reply:
Good flag, done.